danieldupriest/reimbursinator
# Added rules to section and comment backend code

Rules are now returned for each section update. You can view them inside the json.

To test rules:

1. Start docker and create a new report.
2. In "General Info", enter a destination city other than Timbuktu and see the rule violation.
3. In "Flight Info", enter a fare over $500 to see the rule violation.
4. In "Local Transportation", enter an amount greater than $10 per day to see the rule violation.

Backend code is now all commented with docstring comments.

To test:
1. Open backend files.
2. See those phat comments!
